packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -184,6 +184,15 @@ export class CdkErrorMapper {
       errorName: 'FilePermissionsError',
       classification: 'ERROR',
     },
+    {
+      errorRegex:
+        /EPERM: operation not permitted, rename (?<fileName>.*) → 'C:(.*)synth.lock(.*)/,
```

Comment:
```suggestion
        /EPERM: operation not permitted, rename (?<fileName>(.*)\/synth\.lock\.\S+) → '(.*)\/synth\.lock'/,
```

So that:
1. We can drop `C:` letter (what if project is on `D:`?)
2. File name capture is more precise.
3. Dots are special characters in regex, escaped them.
4. Do we need trailing `(.*)` ?